src/frontend/src/pages/AdminToolsPage/AdminToolRecruitmentTab.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Grid, Typography } from '@mui/material';
+import { NERButton } from '../../components/NERButton';
+import { Milestone } from 'shared/src/types/milestone-types';
+import { useHistoryState } from '../../hooks/misc.hooks';
+import CreateMilestoneFormModal from './RecruitmentConfig/CreateMilestoneFormModal';
+import EditMilestoneFormModal from './RecruitmentConfig/EditMilestoneFormModal';
+import MilestoneTable from './RecruitmentConfig/MilestoneTable';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { useAllMilestones } from '../../hooks/recruitment.hooks';
+
+const AdminToolsRecruitmentTab: React.FC = () => {
+  const [createModalShow, setCreateModalShow] = useHistoryState<boolean>('', false);
+  const [editingMilestone, setEditingMilestone] = useHistoryState<Milestone | undefined>('', undefined);
```

Comment:
I'm realizing this naming scheme we've been using is a bit confusing because it suggests that it's a boolean. Maybe make this `milestoneEditing` or something similar